### PR TITLE
[SCB-1639] the configuration about deploying jars  to snapshot reposi…

### PR DESCRIPTION
**Pull request by MabinGo**
the ci configuration about deploying jars  to snapshot repository

### CODE REVIEW

File: scripts/deploy.sh [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+## Unless required by applicable law or agreed to in writing, software
+## distributed under the License is distributed on an "AS IS" BASIS,
+## WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+## See the License for the specific language governing permissions and
+## limitations under the License.
+## ---------------------------------------------------------------------------
+#bin/sh
+
+echo "Deploy a Non-Signed Staging Release"
+mvn deploy -Dorg.slf4j.simpleLogger.log.org.apache.maven.cli.transfer.Slf4jMavenTransferListener=warn -B -DskipTests --settings .travis.settings.xml
```
Comment (chanjarster) [inline scripts/deploy.sh]: I guess you want to suppress the log, I suggest `mvn -q`

Comment (MabinGo) [reply scripts/deploy.sh]: Thanks for your reminder, the property `org.slf4j.simpleLogger.log.org.apache.maven.cli.transfer.Slf4jMavenTransferListener` 
is only used to get rid of the "Downloading/Downloaded" lines, I think the other logs needs to be retained to view the execution track

Comment (chanjarster) [reply scripts/deploy.sh]: Ok, we will keep it that way

File: .travis.settings.xml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+<settings xmlns="http://maven.apache.org/SETTINGS/1.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/SETTINGS/1.0.0 http://maven.apache.org/xsd/settings-1.0.0.xsd">
+  <servers>
+    <server>
+      <id>apache.snapshots.https</id>
+      <username>${env.SONATYPE_USERNAME}</username>
+      <password>${env.SONATYPE_PASSWORD}</password>
+    </server>
+    <server>
+      <id>apache.releases.https</id>
```
Comment (WillemJiang) [inline .travis.settings.xml]: We could remove the release repo setting here, as we don't do release through travis CI.

Comment (kakulisen) [reply .travis.settings.xml]: Ok, I have removed it in the new commit

Comment (MabinGo) [reply .travis.settings.xml]: done

**Comment by chanjarster** [comment]
LGTM

**Pull request merged**